### PR TITLE
Update the description of the Nedelec element.

### PR DESCRIPTION
**Pull request by bangerth**
In the same spirit as #8844.

### CODE REVIEW

**Comment by dangars** [comment]
Very clear!

File: include/deal.II/fe/fe_nedelec.h [hunk 1]
```diff
@@ -42,14 +42,52 @@ DEAL_II_NAMESPACE_OPEN
...
+ * @ref vector_valued "vector-valued", (ii) the shape functions are
+ * discontinuous, but (iii) the tangential component(s) of the vector field
+ * represented by each shape function are continuous across the faces
+ * of cells.
+ *
+ * Other properties of the N&eacute;d&eacute;lec element are that (i) it is
+ * @ref GlossPrimitive "not a primitive element"; (ii) the shape functions
+ * are defined so that certain integrals over the faces are either zero
+ * or one, rather than the common case of certain point values being
+ * either zero or one. (There is, however, the FE_RaviartThomasNodal
```
Comment (kronbichler) [inline include/deal.II/fe/fe_nedelec.h]:
```suggestion
 * either zero or one.
```
I think you do not want to refer to RT here - do we happen to have a nodal H^curl element to mention here?

Comment (bangerth) [reply include/deal.II/fe/fe_nedelec.h]: No, it's really just a copy-paste error.

**Comment by bangerth** [comment]
I fixed the one issue. @kronbichler .

**Pull request merged**